logstash/tests/logstash_test.py

```diff
@@ -571,3 +571,18 @@ def test_adding_a_service():
     assert len(s['spec']['ports']) == 1
     assert s['spec']['ports'][0] == {
         'name': 'beats', 'port': 5044, 'protocol': 'TCP', 'targetPort': 5044}
+
+
+def test_setting_fullnameOverride():
+    config = '''
+fullnameOverride: 'logstash-custom'
+'''
+    r = helm_template(config)
+
+    custom_name = 'logstash-custom'
+    assert custom_name in r['statefulset']
+    assert custom_name in r['service']
```

Comment:
This test ([elastic+helm-charts+pull-request+template-testing/333/CHART=logstash](https://devops-ci.elastic.co/job/elastic+helm-charts+pull-request+template-testing/333/CHART=logstash,label=docker&&virtual/console))is failing because Logstash doesn't have any service with default value.